README.md

```diff
@@ -1,13 +1,116 @@
 # 💪 strong-config
 
+<!-- markdownlint-disable line-length -->
+[![pipeline status](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/pipeline.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master) [![coverage report](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/coverage.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+<!-- markdownlint-enable line-length -->
+
 Secret management for NodeJS. Based on [SOPS](https://github.com/mozilla/sops).
 
-[![pipeline status](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/pipeline.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+Have you ever pushed secrets to your git remote by accidentially committing
+the `.env` file? Then `strong-config` (built by Brickblock) is here to help:
 
-[![coverage report](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/coverage.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+- Manage configs for multiple runtime environments, e.g. different clusters
+- Encrypt secrets at rest and automatically decrypt them at runtime
+- Validate configs against a custom schema
+- For Typescript users: automatically generate types based on schema
 
 ## How-to
 
-Set the environment variable `RUNTIME_ENVIRONMENT`
+1. Install `strong-config` in your project
+
+    ```sh
+    npm install @strong-config/node
+    yarn add @strong-config/node
+    ```
+
+1. Define runtime environment and create first config file
+
+    `strong-config` uses an environment variable to determine which config file
+    to load. By default, this is `RUNTIME_ENVIRONMENT`.
+
+    ```sh
+    export RUNTIME_ENVIRONMENT=development # or similar, depending on your shell
+
+    # in project root
+    mkdir config
```

Comment:
```suggestion
# Create a ./config folder in the root of your project
mkdir config
```

Reply:
IMO too much. Someone worrying about config+secrets management in NodeJS _does_ know what `mkdir` does.

My style is compact here. The less text the better in the first `How-to`.